ironfish/src/primitives/blockheader.ts

```diff
@@ -330,7 +329,7 @@ export class BlockHeaderSerde<
     // as it can be from untrusted sources
     const header = new BlockHeader(
       this.strategy,
-      BigInt(data.sequence),
+      Number(data.sequence),
```

Comment:
we should convert SerializedBlockHeader.sequence to a number now too that it isnt a bigint